core/domain/user_services.py

```diff
@@ -1906,16 +1905,15 @@ def allow_user_to_review_voiceover_in_language(user_id, language_code):
     Args:
         user_id: str. The unique ID of the user.
         language_code: str. The code of the language. Callers should ensure that
-            the user does not have rights to review voiceovers in the given
+            the user does not has rights to review voiceovers in the given
```

Comment:
Ditto.

(Note that "already has" above is correct. But "has" is wrong here.)

Reply:
Done!